tests/python/integrationtest/test_layers.py

```diff
@@ -0,0 +1,92 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+"""Tests for individual operators"""
+# pylint: disable=import-error,no-self-use
+
+from __future__ import absolute_import
+import sys
+import os
+import unittest
+import numpy as np
+import numpy.testing as npt
+from onnx import helper
+import backend as mxnet_backend
+curr_path = os.path.dirname(os.path.abspath(os.path.expanduser(__file__)))
+sys.path.insert(0, os.path.join(curr_path, '../unittest'))
+from common import with_seed
+
+class TestLayers(unittest.TestCase):
+    """Tests for different layers comparing output with numpy operators.
+    Temporary file until we have a corresponding test in onnx-backend_test
+    for these operators."""
+
+    @with_seed(0)
```

Comment:
Please don't use a default seed. (applies to all with_seeds)

Reply:
will use with_seed() instead of the default 0 value.